### PR TITLE
Updating to latest openshift/api and go 1.16.

### PR DESCRIPTION
**Pull request by kwoodson**
Updating the openshift/api as well as go-lang 1.16.

### CODE REVIEW

**Comment by kwoodson** [comment]
@menglingwei @xiaobing.meng Would you mind reviewing this? This bumps the openshift/api dependencies as well as moving to go-lang 1.16.

**Comment by kwoodson** [comment]
@joelspeed PTAL

File: Dockerfile.rhel [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.8 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.9 AS builder
```
Comment (JoelSpeed) [inline Dockerfile.rhel:1]: Should we move this to 4.10?
```suggestion
FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.10 AS builder
```

**Comment by menglingwei** [comment]
> @menglingwei @xiaobing.meng Would you mind reviewing this? This bumps the openshift/api dependencies as well as moving to go-lang 1.16.

Sorry, 9.19-9.21 I'm  on a national holiday. Now i'm going to do it

**Comment by menglingwei** [comment]
> @menglingwei Can we get this merged please?

DONE

**Pull request merged**